README.md

```diff
@@ -107,39 +107,39 @@ Engineering Quality Solutions by employing technologies with Passion and Love |
 <div align="left">
 
 <a href="https://solguruz.com/" target="_blank">
-    <img src="https://github.com/solguruz/multi_selection_filter/blob/feat/modify_readme_file/.github/solguruz.svg" alt=solguruz style="margin-bottom: 5px;" />
+    <img src="https://raw.githubusercontent.com/solguruz/multi_selection_filter/c4c248c7cfafbc5351a97d588c0abff582d01190/.github/solguruz.svg" alt=solguruz style="margin-bottom: 5px;" />
 </a>
```

Comment:
image source should be from main branch instead of commit hash

Reply:
Also please add proper alternate text like SolGuruz on LinkedIn instead of just solguruz